plugins/ember-cli/README.md

```diff
@@ -1,22 +1,24 @@
 # Ember-cli
 
-**Maintainer:** [BilalBudhani](http://www.github.com/BilalBudhani)
+**Maintainer:*[BilalBudhani](http://www.github.com/BilalBudhani)
```

Comment:
Keep the current asterisk, _i.e._ don't change the line. This is how it should look:
![capture20150903150442226](https://cloud.githubusercontent.com/assets/1441704/9658912/373e5694-524d-11e5-8be4-6c94d9836fdc.png)
This is how it looks with your change:
![capture20150903150457561](https://cloud.githubusercontent.com/assets/1441704/9658915/38f43eae-524d-11e5-86bc-b342e06f4beb.png)